lib/grape-swagger/endpoint.rb

```diff
@@ -103,18 +103,28 @@ def path_item(routes, options)
 
     def method_object(route, options, path)
       method = {}
+      method[:summary]     = summary_object(route)
       method[:description] = description_object(route, options[:markdown])
       method[:produces]    = produces_object(route, options[:produces] || options[:format])
       method[:consumes]    = consumes_object(route, options[:format])
       method[:parameters]  = params_object(route)
       method[:responses]   = response_object(route, options[:markdown])
       method[:tags]        = tag_object(route, options[:version].to_s)
+      method[:deprecated]  = deprecated_object(route)
       method[:operationId] = GrapeSwagger::DocMethods::OperationId.build(route, path)
       method.delete_if { |_, value| value.blank? }
 
       [route.request_method.downcase.to_sym, method]
     end
 
+    def summary_object(route)
+      summary = route.options[:desc] if route.options.key?(:desc)
+      summary = route.description if route.description.present?
```

Comment:
I'm also open for proposals 😉 

want say I'm fine with your suggestion above,
maybe you could split of the PR into the `deprecation` stuff and `summary`/`description`…

I want only one point mention, yeap swagger-ui is changing a lot at the moment,
that's why it should be used for view stuff, but not for checking of validity of the generated swagger,
in normal cases, one should prefer http://bigstickcarpet.com/swagger-parser/www/index.html, over it, or one should use [ngrok](https://ngrok.com) (or similar), then this would also be validated

cause IMO a valid swagger is much more important then a nice looking one